user-service/model/user-orm.js

```diff
@@ -39,6 +42,12 @@ export async function isExistingUser(username) {
     return user ? true : false;
 }
 
+// Checks if email exists in database
+export async function isExistingEmail(email) {
+    const user = await getUserByEmail(email);

```

Comment:
Can I check, since the above code converts emails to lowercase, what will happen if there are 2 emails the same when they are lower case (eg. docterjason@gmail.com & DocterJason@gmail.com) when you run getUserByEmail(email)?

Reply:
Good catch! It will not be able to detect that it's a duplicate. I will change it and merge the branch, thanks 😄